src/huggingface_hub/hf_api.py

```diff
@@ -4476,8 +4476,7 @@ def preupload_lfs_files(
             token=token,
         ).xet_enabled
         has_binary_data = any(
-            isinstance(addition.path_or_fileobj, (bytes, io.BufferedIOBase))
-            for addition in new_lfs_additions_to_upload
+            isinstance(addition.path_or_fileobj, (io.BufferedIOBase)) for addition in new_lfs_additions_to_upload
```

Comment:
```suggestion
            isinstance(addition.path_or_fileobj, io.BufferedIOBase) for addition in new_lfs_additions_to_upload
```

(no need for the parenthesis)